embed/src/app.py

```diff
@@ -123,8 +129,8 @@ def get_model(model: str):
     return model_db.get_model_info(model)
 
 
-@app.post("/models/{model}/embed", response_model=EmbeddingResponse, tags=["models"])
-def embed_sentences(model: str, sentences: list[str]):
+@app.post("/models/{model}/embed", tags=["models"])
+def embed_sentences(model: str, sentences: list[str]) -> EmbeddingResponse:
```

Comment:
I have made the un-decorated function explicitly return `EmbeddingResponse` to help the case where this is called directly by `ingest_file`